repos/rust-lang/triagebot.toml

```diff
@@ -7,6 +7,7 @@ bots = ["rustbot"]
 [access.teams]
 infra = "maintain"
 triagebot = "maintain"
+rustdoc = "maintain"
```

Comment:
Sorry, I think there was some confusion. I meant add the new rustdoc directory to https://github.com/rust-lang/rust-forge/blob/648ca186b757f85e556d52a672b10af86c02110e/triagebot.toml#L20 with the appropriate reviewers.

Reply:
Makes more sense haha